### PR TITLE
pkg: use case insensitive event in binlog filter

### PR DESCRIPTION
**Pull request by amyangfei**
### What problem does this PR solve? 

binlog event is lowercase only now. It is not harmful to be case insensitive.

### What is changed and how it works?

change the input rule to lowercase EventType before match.

### Check List 

Tests 

 - Unit test

### CODE REVIEW

**Comment by amyangfei** [comment]
PTAL @GregoryIan @csuzhangxc

File: pkg/binlog-filter/filter.go [hunk 1]
```diff
@@ -265,7 +265,9 @@ func (b *BinlogEvent) Filter(schema, table string, event EventType, rawQuery str
 }
 
 func (b *BinlogEvent) matchEvent(tp, event EventType, rules []EventType) bool {
-	for _, rule := range rules {
+	for _, rawRule := range rules {
+		rule := EventType(strings.ToLower(string(rawRule)))
```
Comment (csuzhangxc) [inline pkg/binlog-filter/filter.go]: can we do this transformation when adding/updating rules?

**Review by csuzhangxc — COMMENTED** [review]
LGTM

**Review by WangXiangUSTC — APPROVED** [review]
LGTM

**Pull request merged**